python/cuml/linear_model/elastic_net.pyx

```diff
@@ -275,3 +280,16 @@ class ElasticNet(Base, RegressorMixin):
                 setattr(self, key, value)
 
         return self
+
+    def __getstate__(self):
```

Comment:
I think this is inherited from Base and done for you, right? Did you add it because the pickle test was failing? Hopefully we don't need it each time...

Reply:
The pickle test was indeed failing but let me give it a shot again. In that case, i won't need a setstate too?

Reply:
John is right, we don't need to add this. At worst if there is a need for a set/getstate in an estimator, it should call Base's set/getstate and then do its logic if I'm not mistaken

Reply:
Removed it, pickle test may have failed for some other reason then which I may have added to the duplicated code.